src/main/electron/use-dark-colors.injectable.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { observable } from "mobx";
+import nativeThemeInjectable from "./native-theme.injectable";
+
+const useDarkColorsInjectable = getInjectable({
+  id: "use-dark-colors",
+  instantiate: (di) => {
+    const nativeTheme = di.inject(nativeThemeInjectable);
+    const state = observable.box(nativeTheme.shouldUseDarkColors);
+
+    nativeTheme.on("updated", () => state.set(nativeTheme.shouldUseDarkColors));
+
+    return state;
```

Comment:
How does it break SRP? Its single responsibility is to be an up to date, observable, version of `nativeTheme.shouldUseDarkColors`.

Reply:
Thoughts:

1. Something called `useDarkColorsInjectable`, should return a function. This returns a state, and therefore it should be called `darkColorsAreUsedStateInjectable`. However, this should probably not return state because...
2. Is it required to expose a way to set this state, if there already is an event (`nativeTheme.on("updated"`) which sets it? These settings of state will compete with each other causing confusion. As a fix, `darkColorsAreUsedInjectable` should be `IComputedValue<boolean>` instead of `IObservableValue<boolean>`.
3. As an improvement, if rigging of `nativeTheme.on("updated")` is yet made a separate responsibility (e.g. `startNativeThemeSyncInjectable`), we could also have another responsibility to clean it up (e.g. `stopNativeThemeSyncInjectable`), which currently does not get done at all. Important: #5324 already does this (see `operatingSystemThemeInjectable`), however the PR is not merged yet, because it is being blocked by #5195 (@Nokel81).

Reply:
I'll clean this up after we merge #5324  then